src/ducks/authentication/MobileRouter.jsx

```diff
@@ -64,6 +86,7 @@ const withAuth = Wrapped => (props, { store }) => {
       }
     }
 
+    cozyClient.login()
     await registerPushNotifications(cozyClient, clientInfos)
```

Comment:
Thought: Push notifications could be extracted from cozy-banks as a plugin for cozy-client